util/config.go

```diff
@@ -55,6 +63,14 @@ var Config = Configuration{
 	UseSSL:           false,
 	ShortedIDLength:  4,
 	AuthBackend:      "oauth",
+	Redis: redisConf{
+		Host:         "127.0.0.1:6379",
+		Password:     "",
+		Db:           0,
```

Comment:
`Db` should be renamed to `DB` because it's a acronym like HTML or XML.

Reply:
Done.